### PR TITLE
common: translate shortened link titles

### PR DESCRIPTION
**Pull request by michamos**
Addresses part of #58.

Signed-off-by: Micha Moskovic <michamos@gmail.com>

### CODE REVIEW

File: inspire_dojson/common/rules.py [hunk 1]
```diff
@@ -53,6 +53,532 @@
...
+    'VACUA': 'Vacuum Server',
+    'VIDEO': 'Watch the video of the talk',
+    'WORLDSCI': 'World Scientific Books Server',
+    'ZBLATT': 'Zentralblatt MATH Server',
+    'ZFDOF': 'Journal of Physical Studies Server',
+    'ZNTFA': 'Z. Naturforsch. Server',
+    'ZPA': 'Z. Phys. A Server',
+    'ZPC': 'Z. Phys. C Server',
+    'FERMILABR': 'Fermilab Library Server (fulltext available)',
+}
```
Comment (jacquerie) [inline inspire_dojson/common/rules.py]: Do we know if this is the latest KB available, or if updates were made to it on Legacy?

CC: @kaplun @tsgit

Comment (michamos) [reply inspire_dojson/common/rules.py]: @kaplun can you detail the one entry that has been added?

Comment (tsgit) [reply inspire_dojson/common/rules.py]: ['ECONFPDF---pdf from eConf']

the live kbr in db has 524 entries, the file in git 
https://github.com/inspirehep/inspire/blob/master/kbs/weblinks.kb
has 523 entries. the difference is the entry above

Comment (tsgit) [reply inspire_dojson/common/rules.py]:
```
In [2]: strings = []

In [3]: with open('kbs/weblinks.kb', 'r') as fh:
   ...:     for line in fh:
   ...:         strings.append[line.rstrip()]

In [5]: len(strings)
Out[5]: 523

In [6]: from invenio.bibknowledge import get_kbr_items

In [7]: livelist = []

In [8]: for x in get_kbr_items('WEBLINKS'):
   ...:     livelist.append("%s---%s" % (x['key'],x['value']))
   ...:     

In [9]: len(livelist)
524

In [10]: [s for s in livelist if s not in strings]
['ECONFPDF---pdf from eConf']

```

Comment (kaplun) [reply inspire_dojson/common/rules.py]: Yeah exactly. Sorry, I have forgot to update the entry here.

**Pull request merged**